### DISCUSSION TITLE
Option to store transcoded videos in a separate location

### DISCUSSION THREAD

**Discussion by SinTan1729** [photoprism/photoprism#2752] [Ideas]
The main reason for me is that I keep everything related to PhotoPrism on my ssd, but the transcoded videos are taking too much space. Since I use them relatively rarely, I think I can get away with storing them in HDD while losing little to no performance.

It can be partially achieved right now by mapping the sidecar for my videos directory to that separate folder. But many of my collections have both photos and videos, so there comes an issue. It'd be great if there were an option to save all `*.avc` files to a different location. I'm not sure how symlinks work inside Docker, but a normal symlink clearly doesn't work. If anyone has any ideas to achieve this with the current state of the application, please do share.

**Comment by lastzero** [comment]
You are right in that symlinks are not supported in the storage folder at this time: https://docs.photoprism.app/known-issues/#symbolic-links

Having more control over the storage locations of generated files is generally useful and should not be particularly difficult to implement. It mainly requires users who are comfortable with additional configuration options and providing better documentation, which is why we're being cautious here. Feel free to submit a [feature request](https://github.com/photoprism/photoprism/issues) for this so we can hopefully provide a solution in a future release!

Thanks for starting a discussion before opening an issue! 😊
